env_shell_python_git/environment.yml

```diff
@@ -0,0 +1,8 @@
+name: swc-shell-python-git
+channels:
+  - swc
+dependencies:
+  - swc-shell
+  - swc-git
+  - swc-python
+  - python=2.7.*
```

Comment:
The lessons are now Python 3. See here:
http://software-carpentry.org/lessons.html

And the official policy, correctly, is that we should teach Python 3. It would be a tragedy to train a whole new generation in Legacy Python! Please push for Py3 at your workshop!